examples/dsk_LoRaE5_Debuging/dsk_LoRaE5_Debuging.ino

```diff
@@ -36,7 +36,7 @@ void loop() {
   static uint8_t data[] = { 0x01, 0x02, 0x03, 0x04 }; 
 
   if ( lorae5.send_sync(
-        1,              // LoRaWan Port
+        8,              // LoRaWan Port
```

Comment:
No reason for using port 8, compared to 1 in the example. 1 is more standard by default